src/main/java/seedu/bankwithus/Parser.java

```diff
@@ -19,15 +19,14 @@ public float parseWithdrawAmt(String args) throws NegativeAmountException {
         if (withdrawAmt < 0) {
             throw new NegativeAmountException();
         }
-        float currBal = bwu.accounts.accounts.get(0).balance;
+        float currBal = bwu.getAccountList().accounts.get(0).balance;
```

Comment:
Wait doesn't the getCurrentAccount method that you added do the exact same thing as accounts.get(0)? I think you can use that method instead